curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/replace-loops-using-recursion.english.md

```diff
@@ -0,0 +1,106 @@
+---
+id: 5cfa3679138e7d9595b9d9d4
+title: Replace Loops using Recursion
+challengeType: 1
+videoUrl: 'https://www.freecodecamp.org/news/how-recursion-works-explained-with-flowcharts-and-a-video-de61f40cb7f9/'
+---
+
+## Description
+<section id='description'>
+Recursion is the concept that a solution can be expressed in terms of itself. To help understand this, start by thinking about the following task: multiply the 0th to nth elements of an array. Using a <code>for</code> loop, you could do this:
```

Comment:
There really is no such thing as the 0th element.  There is an element at index 0, but that is different.
```suggestion
Recursion is the concept that a solution can be expressed in terms of itself. To help understand this, start by thinking about the following task: multiply the first <code>n</code> elements in an array to create the product of the elements. Using a <code>for</code> loop, you could do this:
```